### PR TITLE
Add example for New-AzureADServicePrincipalPasswordCredential

### PR DESCRIPTION
**Pull request by matt40k**
Add example for New-AzureADServicePrincipalPasswordCredential

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[fa803cd](https://github.com/matt40k/azure-docs-powershell-azuread/commits/fa803cd2053d3e0ad3e0f6ccb907cd5db8e0fe73)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md](https://github.com/matt40k/azure-docs-powershell-azuread/blob/patch-1/azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceprincipalpasswordcredential?view=azureadps-2.0&branch=pr-en-us-794) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C15%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202203151547067643-794%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-15T15%3A48%3A25Z&sr=b&sp=r&sig=3kO1fDL65F%2BpVJq5FFZ3o4jTcSdHfk%2BeYMiO1sR%2BzRE%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Hello @matt40k , thanks for your contribution. Please redo this PR with main as the base branch instead of live. Thanks.

**Comment by billmath** [comment]
Docs Build status updates of commit _[fa803cd](https://github.com/matt40k/azure-docs-powershell-azuread/commits/fa803cd2053d3e0ad3e0f6ccb907cd5db8e0fe73)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md](https://github.com/matt40k/azure-docs-powershell-azuread/blob/patch-1/azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceprincipalpasswordcredential?view=azureadps-2.0&branch=pr-en-us-794) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C16%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202203161428245832-794%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-16T14%3A29%3A49Z&sr=b&sp=r&sig=nIGtIOPtiVmLaEAKOxUgh0x8BxC8XMjWKyUf%2FgH9F5E%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md [hunk 1]
```diff
@@ -25,10 +25,12 @@ The New-AzureADServicePrincipalPasswordCredential cmdlet creates a password cred
 
 ### Example 1
 ```powershell
-PS C:\> {{ Add example code here }}
+PS C:\> $endDate = (Get-Date).AddYears(1)
+PS C:\> New-AzureADServicePrincipalPasswordCredential -ObjectId $sp.ObjectId -EndDate $endDate
 ```
 
-{{ Add example description here }}
+This command creates a password credential for a service principal that expires in 1 year
```
Comment (msewaweru) [inline azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md:32]:
```suggestion
This command creates a password credential for a service principal that expires in 1 year.
```

**Comment by msewaweru** [comment]
Thanks @matt40k for your contribution.

@CelesteDG, this is good to go.

**Comment by billmath** [comment]
Docs Build status updates of commit _[8136568](https://github.com/matt40k/azure-docs-powershell-azuread/commits/813656855f01b198712c8d330ca634c4b6d056b9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md](https://github.com/matt40k/azure-docs-powershell-azuread/blob/patch-1/azureadps-2.0/AzureAD/New-AzureADServicePrincipalPasswordCredential.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/new-azureadserviceprincipalpasswordcredential?view=azureadps-2.0&branch=pr-en-us-794) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C17%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202203170928279061-794%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-17T09%3A29%3A31Z&sr=b&sp=r&sig=lqQnK6tMGJ9jZtyYjIlLh1IrqxvZ8skT%2BQ9fcraLHcM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**